services/Doordash.mjs

```diff
@@ -188,5 +188,35 @@ class Doordash extends Service {
       throw new HTTPResponseError(res);
     }
   }
+  async getStore(storeID) {
```

Comment:
Lets add some documentation to this comment with a description of what it does and @params for the type of expected input data as well as @return so we know what is returned.

Reply:
added!